nautobot_ssot_aristacv/diffsync/models/nautobot.py

```diff
@@ -112,6 +113,11 @@ def create(cls, diffsync, ids, attrs):
     def update(self, attrs):
         """Update Interface in Nautobot."""
         _port = OrmInterface.objects.get(id=self.uuid)
+        if "description" in attrs:
+            description = ""
```

Comment:
More for my learning, but why do you set description to an empty string here?

Reply:
This allows for the case of when a description is removed from a Port in CVP, it is then updated to be blank in Nautobot. The attribute has to be a string and can't be None.